### DISCUSSION TITLE
CRTouch failing after 67 probes

### DISCUSSION THREAD

**Discussion by cds333** [mriscoc/Ender3V2S1#254] [Q&A]
Using Ender3V2-427-BLT configuration on an ender 3; whenever I set it to 9x9 it fails every time on point 67 and restarts the leveling. Gets to point 67 again and repeats. It never finishes. I have the exact same firmware with Ender3V2-422-BLT config on an ender3v2 and the 9x9 is working perfectly.

On the 4.2.7 board I can only set a max of 8x8 or it will fail at point 67. 

One thing that should not matter - the 4.2.2 is using a BLTouch while the 4.2.7 is using a CRTouch.

Please advise.

**Comment by mriscoc** [comment]
Check notes about CRTouch: https://github.com/mriscoc/Ender3V2S1/wiki/3D-BLTouch#notes-about-crtouch

**Reply by cds333** [reply]
Much appreciated. Thank you sir! I will make the recommended corrections.

But do we know why? 

Why would it consistently fail at the same number of probes?

And why does the BL not do this?
